modules/contact/test/tests/bouncing-block-contact/grid-sequencing/tests

```diff
@@ -23,5 +23,6 @@
     partial = True
     mesh_mode = replicated # Fix when mortar segment mesh is distributed
     rel_err = 2e-5
+    allow_warnings = True # This is a difficult problem that can result in poor mortar projections that emit warnings
```

Comment:
This seems unrelated, why did you encounter this issue in this context?

Reply:
Hmm, it looks like I picked up a stray commit somehow. I'll fix that.